README.md

```diff
@@ -17,3 +18,14 @@ Please contact your technical support representative for any questions about the
 
 ## License
 [Apache License v2.0](LICENSE)
+
+## Contributing
+
+Any contribution is welcome, as an issue or pull request. 
+
+When contributing to this repository, if the changes are not trivial (such as typos or obvious programming mistakes),
```

Comment:
Slightly confusing as I was expecting the example to be related to non trivial changes. I would reword to: "unless the changes are trivial (typos, obvious programming mistakes..), we encourage creating an issue first, to discuss the change you wish to make.

PS: As far as contributing is concerned, I am omitting other channels of communication.

Reply:
Done